internal/pkg/sessions/cookie_store.go

```diff
@@ -167,7 +171,7 @@ func (s *CookieStore) LoadSession(req *http.Request) (*SessionState, error) {
 	}
 	session, err := UnmarshalSession(c.Value, s.CookieCipher)
 	if err != nil {
-		return nil, err
```

Comment:
Let's add some logging for this error and/or create some metrics for it.

I don't want to just swallow this error as it may be very useful for debugging context, despite not being useful further up the stack.